### DISCUSSION TITLE
Open multiple windows in native app

### DISCUSSION THREAD

**Discussion by mbolder** [zauberzeug/nicegui#1101] [Ideas / Feature Requests]
### Question

Hello,
is there a way to create a new / second window in native mode ?
I would like to use more windows like a dialog, but without stopping the main window.

**Comment by rodja** [comment]
A similar question has been asked on [Discord a few days back](https://discord.com/channels/1089836369431498784/1119593174634070038). Unfortunately, NiceGUI only supports one main window right now. The main problem is, that the windows are managed with PyWebview which requires running in a separate process. So we need to proxy all commands from NiceGUI to PyWebview. You can have a look at the app.native.main_window implementation. We would be very happy if someone could extend this...
